src/wrap.js

```diff
@@ -29,7 +29,7 @@ var arity = require('./arity');
  *
```

Comment:
Hmm, no example right now. Will do a PR if any come to mind. Thanks for the fix.